CODEOWNERS

```diff
@@ -1,46 +1,97 @@
+apps/ledger-live-desktop/src/renderer/reducers/swap.ts          @ledgerhq/ptx
+apps/ledger-live-desktop/src/renderer/screens/earn/             @ledgerhq/ptx
+apps/ledger-live-desktop/src/renderer/screens/exchange/         @ledgerhq/ptx
+apps/ledger-live-desktop/src/renderer/screens/stake/            @ledgerhq/ptx
+apps/ledger-live-desktop/src/renderer/screens/swapWeb/          @ledgerhq/ptx
+apps/ledger-live-mobile/src/actions/earn.ts                     @ledgerhq/ptx
+apps/ledger-live-mobile/src/actions/swap.ts                     @ledgerhq/ptx
+apps/ledger-live-mobile/src/components/ProviderIcon/            @ledgerhq/ptx
+apps/ledger-live-mobile/src/components/Stake/                   @ledgerhq/ptx
+apps/ledger-live-mobile/src/components/WebPTXPlayer/            @ledgerhq/ptx
+apps/ledger-live-mobile/src/reducers/earn.ts                    @ledgerhq/ptx
+apps/ledger-live-mobile/src/reducers/swap.ts                    @ledgerhq/ptx
+apps/ledger-live-mobile/src/screens/Exchange/                   @ledgerhq/ptx
+apps/ledger-live-mobile/src/screens/PTX/                        @ledgerhq/ptx
+apps/ledger-live-mobile/src/screens/Swap/                       @ledgerhq/ptx
+libs/ledger-live-common/src/exchange/                           @ledgerhq/ptx
 
-# Wallet API
-ledger-live-common/src/platform/                    @LedgerHQ/wallet-api
+# Wallet API team
+**/PlatformAppProviderWrapper.tsx                        @ledgerhq/wallet-api
+**/Web3AppWebview/                                       @ledgerhq/wallet-api
+**/WebPlatformPlayer/                                    @ledgerhq/wallet-api
+apps/ledger-live-desktop/src/renderer/screens/platform/  @ledgerhq/wallet-api
+apps/ledger-live-mobile/src/screens/Discover/            @ledgerhq/wallet-api
+libs/ledger-live-common/src/platform/                    @ledgerhq/wallet-api
+libs/ledger-live-common/src/wallet-api/                  @ledgerhq/wallet-api
+libs/test-utils/                                         @ledgerhq/wallet-api
 
-# Live Devices Team
-apps/**/components/DeviceAction/        @ledgerhq/live-devices
-ledger-live-mobile/src/screens/Manager/ @ledgerhq/live-devices
-ledger-live-common/src/hw/              @ledgerhq/live-devices
-ledger-live-common/src/apps/            @ledgerhq/live-devices
-ledger-live-common/src/manager/         @ledgerhq/live-devices
-ledgerjs/packages/types-devices/        @ledgerhq/live-devices
-ledgerjs/packages/devices/              @ledgerhq/live-devices
-ledgerjs/packages/hw-transport-*        @ledgerhq/live-devices
-ledgerjs/packages/react-native-*        @ledgerhq/live-devices
+# Devices team
+**/src/renderer/screens/manager/                         @ledgerhq/live-devices
+apps/web-tools/repl/                                     @ledgerhq/live-devices
+apps/**/components/DeviceAction/                         @ledgerhq/live-devices
+ledger-live-mobile/src/screens/Manager/                  @ledgerhq/live-devices
```

Comment:
[SHOULD] path should be `apps/ledger-live-mobile/...` ?

Reply:
```
libs/**/devices/
```

will match

libs/ledgerjs/packages/devices/

but later, we will move it to libs/devices so i'm anticipating this a bit.